vllm/worker/hpu_model_runner.py

```diff
@@ -672,9 +672,9 @@ def _is_valid_bucket(self, bucket):
     def _setup_buckets(self) -> None:
         align_bs = lambda x: min(self.max_num_seqs, x)
         max_bucket_cfg = 64
-        if self.lora_config and \
-            max_bucket_cfg > self.max_num_batched_tokens // self.block_size:
-            max_bucket_cfg = self.max_num_batched_tokens // self.block_size
+        # if self.lora_config and \
```

Comment:
Is this a draft? It contains only commented out code

Reply:
Updated